File: connector/connector-domparser/src/main/java/com/github/thetric/iliasdownloader/service/webparser/IliasItemIdStringParsingException.java
package com.github.thetric.iliasdownloader.service.webparser;

import com.github.thetric.iliasdownloader.service.exception.IliasException;

/**
 * @author broj
 * @since 18.08.2016
 */
public final class IliasItemIdStringParsingException extends IliasException {
    public IliasItemIdStringParsingException(String message) {
        super(message);
    }

    public IliasItemIdStringParsingException(String message, Throwable t) {
        super(message, t);
    }

    public IliasItemIdStringParsingException(Throwable t) {
        super(t);
    }
}

File: src/main/kotlin/com/github/thetric/iliasdownloader/connector/domparser/impl/IliasItemIdStringParsingException.kt
package com.github.thetric.iliasdownloader.connector.domparser.impl

import com.github.thetric.iliasdownloader.connector.api.exception.IliasException

/**
 * Indicates an error when parsing an Ilias item ID string.
 */
class IliasItemIdStringParsingException : IliasException {
    constructor(message: String) : super(message)

    constructor(message: String, t: Throwable) : super(message, t)
}
